tensorbay/cli/branch.py

```diff
@@ -14,13 +14,21 @@
 from .utility import error, get_dataset_client, get_gas
 
 
-def _implement_branch(obj: Dict[str, str], tbrn: str, name: str, verbose: bool) -> None:
+def _implement_branch(
+    obj: Dict[str, str], tbrn: str, name: str, verbose: bool, is_delete: bool
+) -> None:
     info = TBRN(tbrn=tbrn)
-
     if info.type != TBRNType.DATASET:
         error(f'To operate a branch, "{info}" must be a dataset')
 
     gas = get_gas(**obj)
+    if is_delete:
+        branch_name = str(info.revision)
+        info.revision = None
+        dataset_client = get_dataset_client(gas, info)
```

Comment:
These can be moved into `_delete_branch`.
`info.revision` is str or None, `str(info.revison)` here will change `None` to `"None"`.